charts/immich/values.yaml

```diff
@@ -7,7 +7,7 @@
 env:
   REDIS_HOSTNAME: '{{ printf "%s-redis-master" .Release.Name }}'
   DB_HOSTNAME: "{{ .Release.Name }}-postgresql"
-  DB_USERNAME: "{{ .Values.postgresql.global.postgresql.auth.username }}"
+  DB_USERNAME: "postgres"
   DB_DATABASE_NAME: "{{ .Values.postgresql.global.postgresql.auth.database }}"
   # -- You should provide your own secret outside of this helm-chart and use `postgresql.global.postgresql.auth.existingSecret` to provide credentials to the postgresql instance
   DB_PASSWORD: "{{ .Values.postgresql.global.postgresql.auth.password }}"
```

Comment:
I believe this line should be updated to refer to `"{{ .Values.postgresql.global.postgresql.auth.postgresPassword }}"` instead, as this can (and should) be set by the user independently from the `immich` user's password value.

Reply:
Yes, you're right. I missed that field. Will push a fix